lisp/ein-multilang.el

```diff
@@ -142,6 +143,19 @@ This function may raise an error."
   (set-syntax-table python-mode-syntax-table)
   (set-keymap-parent ein:notebook-multilang-mode-map python-mode-map))
 
+(defun ein:ml-lang-setup-clojure ()
```

Comment:
You should make the code here conditional on `(featurep 'clojure-mode)` being true.